decidim-proposals/spec/features/index_proposal_notes_spec.rb

```diff
@@ -0,0 +1,51 @@
+# frozen_string_literal: true
+
+require "spec_helper"
+
+describe "Index Proposal Notes", type: :feature do
```

Comment:
This spec is taking care of both indexing and showing proposal notes, so maybe we should change this to `Manage proposal notes`? It might not look very relevant, but things like this might really confuse future developers :)